README.md

```diff
@@ -4,8 +4,12 @@ This repository includes scripts to set up a Metal³ development environment.
 
 ## Build Status
 
-[![Ubuntu daily main build status](https://jenkins.nordix.org/buildStatus/icon?job=metal3_daily_main_integration_test_ubuntu&subject=Ubuntu%20daily%20main)](https://jenkins.nordix.org/view/Metal3/job/metal3_daily_main_integration_test_ubuntu/)
-[![CentOS daily main build status](https://jenkins.nordix.org/buildStatus/icon?job=metal3_daily_main_integration_test_centos&subject=CentOS%20daily%20main)](https://jenkins.nordix.org/view/Metal3/job/metal3_daily_main_integration_test_centos/)
+[![Ubuntu Integration daily main build status](https://jenkins.nordix.org/buildStatus/icon?job=metal3_daily_main_integration_test_ubuntu&subject=Ubuntu%20daily%20main)](https://jenkins.nordix.org/view/Metal3%20Periodic/job/metal3_daily_main_integration_test_ubuntu/)
+[![CentOS Integration daily main build status](https://jenkins.nordix.org/buildStatus/icon?job=metal3_daily_main_integration_test_centos&subject=CentOS%20daily%20main)](https://jenkins.nordix.org/view/Metal3%20Periodic/job/metal3_daily_main_integration_test_centos/)
+[![Ubuntu Integration daily release-1.2 build status](https://jenkins.nordix.org/buildStatus/icon?job=metal3_daily_release-1-2_integration_test_ubuntu&subject=Ubuntu%20daily%20release-1.2)](https://jenkins.nordix.org/view/Metal3%20Periodic/job/metal3_daily_release-1-2_integration_test_ubuntu/)
+[![CentOS Integration daily release-1.2 build status](https://jenkins.nordix.org/buildStatus/icon?job=metal3_daily_release-1-2_integration_test_centos&subject=CentOS%20daily%20release-1.2)](https://jenkins.nordix.org/view/Metal3%20Periodic/job/metal3_daily_release-1-2_integration_test_centos/)
+[![Ubuntu Integration daily release-1.1 build status](https://jenkins.nordix.org/buildStatus/icon?job=metal3_daily_release-1-1_integration_test_ubuntu&subject=Ubuntu%20daily%20release-1.1)](https://jenkins.nordix.org/view/Metal3%20Periodic/job/metal3_daily_release-1-1_integration_test_ubuntu/)
+[![CentOS Integration daily release-1.1 build status](https://jenkins.nordix.org/buildStatus/icon?job=metal3_daily_release-1-1_integration_test_centos&subject=CentOS%20daily%20release-1.1)](https://jenkins.nordix.org/view/Metal3%20Periodic/job/metal3_daily_release-1-1_integration_test_centos/)
```

Comment:
I dont see the point of showing the release-1.1 builds here though.

Reply:
Okay, I kept only a new `release-1.2` branch status badge on pair with `main`, since that is the latest stable branch and can be kept IMO.